spec/fixtures/reversed_timezone.ics

```diff
@@ -0,0 +1,143 @@
+BEGIN:VCALENDAR
+VERSION:2.0
+X-WR-CALNAME:My Calendar
+X-APPLE-CALENDAR-COLOR:#44A703
+BEGIN:VEVENT
+CREATED:20231026T211915Z
+DTEND;TZID=NONSTANDARDTZ:20231025T180000
+DTSTAMP:20231026T211916Z
+DTSTART;TZID=NONSTANDARDTZ:20231025T163000

```

Comment:
using this `TZID` proves that this solution should work whether or not `tzinfo` or `activesupport` are being used.